### PR TITLE
{data}[foss/2019b] pycocotools v2.0.0 w/ Python 3.7.4

### PR DESCRIPTION
**Pull request by hajgato**

### CODE REVIEW

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
easybuild.novalocal - Linux centos linux 7.8.2003, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 2.7.5
See https://gist.github.com/a9791859b3496ad390013ff3e9f3710f for a full test report.

File: easybuild/easyconfigs/p/pycocotools/pycocotools-2.0.0-foss-2019b-Python-3.7.4.eb [hunk 1]
```diff
@@ -0,0 +1,25 @@
+easyblock = 'PythonPackage'
+
+name = 'pycocotools'
+version = '2.0.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/cocodataset/cocoapi'
+description = "Tools for working with the MSCOCO dataset"
+
+toolchain = {'name': 'foss', 'version': '2019b'}
```
Comment (verdurin) [inline easybuild/easyconfigs/p/pycocotools/pycocotools-2.0.0-foss-2019b-Python-3.7.4.eb:10]: @hajgato This looks fine. The only question is whether it needs MPI/BLAS etc., or whether it should live at a lower toolchain e.g. `GCCcore/8.3.0`.

Comment (hajgato) [reply easybuild/easyconfigs/p/pycocotools/pycocotools-2.0.0-foss-2019b-Python-3.7.4.eb:10]: Well, it needs SciPy, and we do not have that without MPI/BLAS as far as I know.

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node368.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz (skylake_avx512), Python 2.7.5
See https://gist.github.com/06881b4292c791bc7f73ad93e814ce33 for a full test report.

**Comment by smoors** [comment]
Going in, thanks @hajgato!

**Pull request merged**